PhysicsTools/PatAlgos/plugins/PATElectronProducer.cc

```diff
@@ -91,6 +91,7 @@ PATElectronProducer::PATElectronProducer(const edm::ParameterSet & iConfig) :
   embedHighLevelSelection_(iConfig.getParameter<bool>("embedHighLevelSelection")),
   beamLineToken_(consumes<reco::BeamSpot>(iConfig.getParameter<edm::InputTag>("beamLineSrc"))),
   pvToken_(mayConsume<std::vector<reco::Vertex> >(iConfig.getParameter<edm::InputTag>("pvSrc"))),  
+  getdBFromTrack_(iConfig.getParameter<bool>("getdBFromTrack")),
```

Comment:
This change is not present in the master version!
Please clarify

Reply:
Yes, because in the master the dxy sign is "just fixed", while in this case the fix has to be activated by this switch.

Compare #28753 and #29537 where the same difference can be seen.